src/components/AvailabilityGrid.tsx

```diff
@@ -232,8 +233,54 @@ const Grid = ({
     setGrid(newGrid)
   }
 
+  // Function to generate grid gradient based on the number of responders
+  const generateGridGradient = (
+    isSelected: number,
+    totalResponders: number,
+  ): string => {
+    if (totalResponders === 0) return '' // No responders, no color
+
+    // Define shades
+    const shades = [100, 200, 300, 400, 500, 600]
+
+    // Calculate fraction and map to shade index
+    const fraction = isSelected / totalResponders
+    const index = Math.min(
+      Math.floor(fraction * (shades.length - 1)),
+      shades.length - 1,
```

Comment:
I can't rationalize why the min function was needed except if fraction is > 1 which it shouldn't be.  Not a horrible sanity check i guess.